subprojects/delivery/nupokati/src/main/kotlin/com/avito/android/model/CdBuildConfig.kt

```diff
@@ -0,0 +1,45 @@
+package com.avito.android.model
+
+public data class CdBuildConfig(
+    val schemaVersion: Long,
+    val project: NupokatiProject,
+    val outputDescriptor: OutputDescriptor,
+    val releaseVersion: String,
+    val deployments: List<Deployment>
+) {
+
+    public data class OutputDescriptor(
+        val path: String,
+        val skipUpload: Boolean
+    )
+
+    public sealed class Deployment {
+
+        public data class GooglePlay(
+            val artifactType: AndroidArtifactType,
+            val buildVariant: BuildVariant,
+            val track: Track
+        ) : Deployment()
+
+        /**
+         * Deploy artifacts to QApps.
+         * Uses UploadToQapps build step to find them.
+         */
+        public data class Qapps(
+            /**
+             * Send artifacts as release versions.
+             * Non-release artifacts stored limited time.
+             */
+            val isRelease: Boolean
+        ) : Deployment()
+
+        public data class Unknown(val type: String) : Deployment()
+
+        public enum class Track {
```

Comment:
Shouldn't this model be inside Google Play model?
Otherwise, it looks like it can be used inside any deployment.

Reply:
I'd skip it for same reasons: https://github.com/avito-tech/avito-android/pull/1307#discussion_r784019736